src/generic3/templates/deployment.yaml

```diff
@@ -285,7 +285,10 @@ spec:
         - mountPath: {{ .Values.app.volume.mountPath }}
           name: {{ template "generic.name" . }}-data
         {{ end }}
-
+        {{ if .Values.app.volume.existingClaim }}
+        - mountPath: {{ .Values.app.volume.mountPath }}
+          name: pvcname
```

Comment:
Can this value `pvcname` be specified in the `values.yml`

Reply:
we can add it if needed but I don't think it's required only the claimName matters.